src/ColorVectorSpace.jl

```diff
@@ -10,7 +10,7 @@ import Base: abs, abs2, clamp, convert, copy, div, eps, isfinite, isinf,
 import LinearAlgebra: norm
 import StatsBase: histrange, varm
 import SpecialFunctions: gamma, lgamma, lfact
-import Statistics: middle
+import Statistics: middle, var, std
 
 export nan
```

Comment:
`complement` needs to be exported so that users don't need to write `ColorVectorSpaces.complement`

Reply:
Would that be as simple as `export complement` in that case?